tests/backup/backup_helper.go

```diff
@@ -3910,6 +3911,50 @@ func CreateRestoreWithProjectMapping(restoreName string, backupName string, name
 	return nil
 }
 
+// CreateRestoreWithProjectMappingWithoutCheckWithReplacePolicy creates restore with project mapping
+func CreateRestoreWithProjectMappingWithoutCheckWithReplacePolicy(restoreName string, backupName string, namespaceMapping map[string]string, clusterName string,
```

Comment:
Can we simply call this `CreateRestoreOnRancherWithoutCheck` ?

Reply:
done